graphql_client_cli/src/main.rs

```diff
@@ -39,9 +43,15 @@ enum Cli {
     #[structopt(name = "generate")]
     Generate {
         // should be a glob
-        paths: String,
         #[structopt(parse(from_os_str))]
-        schema: PathBuf,
+        query_path: PathBuf,
+        #[structopt(parse(from_os_str))]
+        schema_path: PathBuf,
+        selected_operation: String,
+        #[structopt(short = "a", long = "additional_derives")]
+        additional_derives: Option<String>,
+        #[structopt(short = "d", long = "deprecation_strategy",)]
+        deprecation_strategy: Option<u8>,
```

Comment:
For additional derives and deprecation strategy, I prefer hyphens instead of underscores for the `long` names, so we can write `graphql-client-cli generate --additional-derives=Serialize,Clone` instead of `--additional_derives`.

For the deprecation strategy options could we take a string instead? I think `--deprecation-strategy=allow` is more intuitive than `--deprecation-strategy=0`.

Reply:
OK I'll fix it later :)

Reply:
There's no rush :)